commit 5554cefca5021afa3edc781de5d54019f373cc92
Author: Daniel J Walsh <dwalsh@redhat.com>
Date:   Wed Nov 25 13:41:46 2020 -0500

    Add ability to set system wide options for slirp4netns
    
    Also fixup some formatting issues in man pages.
    
    Fix generation of multiple man pages.
    
    Partial fix for: https://github.com/containers/podman/issues/7888
    
    Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

diff --git a/docs/Makefile b/docs/Makefile
index 4ed5b1c..9be1f61 100644
--- a/docs/Makefile
+++ b/docs/Makefile
@@ -5,7 +5,7 @@ GOMD2MAN ?= $(shell command -v go-md2man || echo '$(GOBIN)/go-md2man')
 
 docs: $(patsubst %.md,%,$(wildcard *.md))
 
-%.5: *md
+%.5:  %.5.md
 	$(GOMD2MAN) -in $^ -out $@
 
 .PHONY: install
